### PR TITLE
GA4 send_to parameter update

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/google-analytics-4/setting-up-google-analytics-4-in-rudderstack.mdx [hunk 1]
```diff
@@ -86,6 +86,12 @@ Google Analytics 4 has a limit on the number of unique properties per event name
   - If enabled, the user ID is set to the identified visitors and sent as explained in the [Cookies and user identification](https://developers.google.com/tag-platform/devguides/cookies#set_user_id) guide.
   - If disabled, `userId` is not sent and RudderStack only sends the user traits set as `user_properties` to Google Analytics. The user properties are set on the `gtag` client as explained in the [Set user properties](https://developers.google.com/analytics/devguides/collection/ga4/user-properties?technology=websites) guide.
 
+- **Add send_to parameter in gtag event call**: Enable this setting to send an event **only** to the specific Google Analytics 4 property whose measurement ID you have mentioned in the connection settings above. When enabled, a sample event to Google Analytics 4 looks like below:
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/google-analytics-4/setting-up-google-analytics-4-in-rudderstack.mdx:89]: Can we add GA4 send_to parameter doc link here
https://support.google.com/analytics/answer/9310895?hl=en&ref_topic=11192706#zippy=%2Cin-this-article

**Review by Gauravudia — APPROVED** [review]

File: docs/destinations/streaming-destinations/google-analytics-4/google-analytics-4-cloud-mode.mdx [hunk 2]
```diff
@@ -604,11 +604,17 @@ You can use the Google Analytics 4 `session_id` parameter to identify the sessio
 To read more about sessions in Google Analytics 4, refer to this <a href="https://support.google.com/analytics/answer/9191807?hl=en#zippy=">Google Analytics 4 Help article</a>.
 </div>
 
-RudderStack maps the following properties to GA4's `session_id` property:
+RudderStack maps the following sessions properties for the `identify` and `group` call:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/google-analytics-4/google-analytics-4-cloud-mode.mdx:607]:
```suggestion
RudderStack maps the following session properties for the `identify` and `group` call:
```

File: docs/destinations/streaming-destinations/google-analytics-4/google-analytics-4-cloud-mode.mdx [hunk 3]
```diff
@@ -604,11 +604,17 @@ You can use the Google Analytics 4 `session_id` parameter to identify the sessio
...
 
 | RudderStack properties | Google Analytics 4 property |
 | :------------------------| :-------------|
-| `traits.sessionId`<br />`context.traits.sessionId`<br />`traits.session_id`<br />`context.traits.session_id` | `session_id` |
+| `traits.sessionId`<br />`context.traits.sessionId`<br />`traits.session_id`<br />`context.traits.session_id` <br />`context.sessionId` | `session_id` |
+
+RudderStack maps the following sessions properties for the `track` and `page` call:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/google-analytics-4/google-analytics-4-cloud-mode.mdx:613]:
```suggestion
RudderStack maps the following session properties for the `track` and `page` calls:
```

File: docs/destinations/streaming-destinations/google-analytics-4/setting-up-google-analytics-4-in-rudderstack.mdx [hunk 4]
```diff
@@ -86,6 +86,13 @@ Google Analytics 4 has a limit on the number of unique properties per event name
...
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/google-analytics-4/setting-up-google-analytics-4-in-rudderstack.mdx:89]:
```suggestion
- **Add send_to parameter in gtag event call**: Enable this setting to send an event **only** to the specific Google Analytics 4 property whose **Measurement Id** you have specified above. When enabled, a sample event to Google Analytics 4 looks as shown:
```

**Pull request merged**